webpack/assets/javascripts/react_app/components/HostDetails/Tabs/Details/Cards/Virtualization/VirtLibvirt.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  DescriptionListTerm,
+  DescriptionListGroup,
+  DescriptionListDescription,
+} from '@patternfly/react-core';
+import { number_to_human_size as NumberToHumanSize } from 'number_helpers';
+import { translate as __ } from '../../../../../../common/I18n';
+
+const VirtLibvirt = ({ vm }) => (
+  <>
+    <DescriptionListGroup>
+      <DescriptionListTerm>{__('Name')}</DescriptionListTerm>
+      <DescriptionListDescription>{vm.name}</DescriptionListDescription>
+    </DescriptionListGroup>
+    <DescriptionListGroup>
+      <DescriptionListTerm>{__('Machine type')}</DescriptionListTerm>
+      <DescriptionListDescription>{`${vm.domain_type}/${vm.arch}`}</DescriptionListDescription>
+    </DescriptionListGroup>
+    <DescriptionListGroup>
+      <DescriptionListTerm>{__('VCPU(s)')}</DescriptionListTerm>
+      <DescriptionListDescription>{vm.cpus}</DescriptionListDescription>
+    </DescriptionListGroup>
+    <DescriptionListGroup>
+      <DescriptionListTerm>{__('UUID')}</DescriptionListTerm>
+      <DescriptionListDescription>{vm.uuid}</DescriptionListDescription>
+    </DescriptionListGroup>
+    <DescriptionListGroup>
+      <DescriptionListTerm>{__('Memory')}</DescriptionListTerm>
+      <DescriptionListDescription>
+        {`${NumberToHumanSize(vm.max_memory_size)} (${__(
+          'Allocated'
+        )}: ${NumberToHumanSize(vm.memory_size)})`}
+      </DescriptionListDescription>
+    </DescriptionListGroup>
+    {vm.nics.map(nic => (
+      <DescriptionListGroup>
+        <DescriptionListTerm>{__('NIC')}</DescriptionListTerm>
+        <DescriptionListDescription>
+          {`${nic.bridge} - ${nic.mac} (${nic.model})`}
+        </DescriptionListDescription>
+      </DescriptionListGroup>
+    ))}
+    {Object.values(vm.volumes_attributes).map((vol, index) => (
+      <DescriptionListGroup key={index}>
+        <DescriptionListTerm>{__('Disk')}</DescriptionListTerm>
+        <DescriptionListDescription>
+          {__(
+            `using ${vol.allocation} GB out of ${vol.capacity} GB, ${vol.pool_name} storage pool`
```

Comment:
This should use `sprintf` https://projects.theforeman.org/projects/foreman/wiki/Translating#JavaScript